CLI/Repl.cpp

```diff
@@ -253,6 +253,41 @@ static void completeRepl(lua_State* L, const char* editBuffer, std::vector<std::
     completeIndexer(L, editBuffer, start, completions);
 }
 
+#ifdef LUAU_WEB_REPL
```

Comment:
Would it make any sense to simplify the configuration by using `__EMSCRIPTEN__` here instead? That way you can just build CLI using Emscripten instead of setting up a whole new target.